config.go

```diff
@@ -232,7 +233,7 @@ func newQemuHypervisorConfig(h hypervisor) (vc.HypervisorConfig, error) {
 		DefaultVCPUs:          h.defaultVCPUs(),
 		DefaultMemSz:          h.defaultMemSz(),
 		DisableBlockDeviceUse: h.DisableBlockDeviceUse,
-		MemPrealloc:           h.MemPrealloc,
```

Comment:
can you expand on why the MemPrealloc change as well pls?

Reply:
@grahamwhaley no, that is a bug. I will fix it. The two are independent.